### PR TITLE
Add TransparentPtr marker trait to List, SList and StrVItem

### PR DESCRIPTION
**Pull request by pbor**

### CODE REVIEW

File: glib/src/collections/strv.rs [hunk 1]
```diff
@@ -11,6 +11,21 @@ use crate::{translate::*, GStr, GString};
...
+
+#[doc(hidden)]
+impl GlibPtrDefault for StrVItem {
+    type GlibType = *mut c_char;
+}
+
+impl Clone for StrVItem {
+    #[inline]
+    fn clone(&self) -> StrVItem {
+        unsafe { StrVItem(ptr::NonNull::new_unchecked(ffi::g_strdup(self.0.as_ptr()))) }
```
Comment (sdroege) [inline glib/src/collections/strv.rs:25]: You also need `Drop` then

Comment (sdroege) [reply glib/src/collections/strv.rs:25]: Might make sense to rename this and give it a constructor too. By having `Clone` we make it proper API :)

Comment (sdroege) [reply glib/src/collections/strv.rs:25]: `GStrPtr`?

Comment (pbor) [reply glib/src/collections/strv.rs:25]: Ok... let me drop it from this PR and open a separate one

**Review by sdroege — APPROVED** [review]

**Pull request merged**